[2022-02-13 17:04:06,466] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_neo4j.neo4j_hook manual__2022-02-13T17:04:02.451248+00:00 [queued]>
[2022-02-13 17:04:06,519] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_neo4j.neo4j_hook manual__2022-02-13T17:04:02.451248+00:00 [queued]>
[2022-02-13 17:04:06,521] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 17:04:06,523] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-13 17:04:06,525] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-13 17:04:06,570] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): neo4j_hook> on 2022-02-13 17:04:02.451248+00:00
[2022-02-13 17:04:06,587] {standard_task_runner.py:52} INFO - Started process 103 to run task
[2022-02-13 17:04:06,605] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_neo4j', 'neo4j_hook', 'manual__2022-02-13T17:04:02.451248+00:00', '--job-id', '270', '--raw', '--subdir', 'DAGS_FOLDER/test_neo4j.py', '--cfg-path', '/tmp/tmpstu9nxub', '--error-file', '/tmp/tmpkh5vkxi_']
[2022-02-13 17:04:06,613] {standard_task_runner.py:77} INFO - Job 270: Subtask neo4j_hook
[2022-02-13 17:04:06,801] {logging_mixin.py:109} INFO - Running <TaskInstance: test_neo4j.neo4j_hook manual__2022-02-13T17:04:02.451248+00:00 [running]> on host 48e82ab1b440
[2022-02-13 17:04:06,909] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_neo4j
AIRFLOW_CTX_TASK_ID=neo4j_hook
AIRFLOW_CTX_EXECUTION_DATE=2022-02-13T17:04:02.451248+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-13T17:04:02.451248+00:00
[2022-02-13 17:04:06,921] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-13 17:04:06,927] {base.py:79} INFO - Using connection to: id: neo4j_conn_id. Host: host.docker.internal, Port: None, Schema: None, Login: neo4j, Password: ***, extra: {}
[2022-02-13 17:04:06,929] {neo4j.py:59} INFO - URI: bolt://host.docker.internal:7687
[2022-02-13 17:04:06,961] {logging_mixin.py:109} INFO - []
[2022-02-13 17:04:06,964] {python.py:175} INFO - Done. Returned value was: None
[2022-02-13 17:04:06,987] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_neo4j, task_id=neo4j_hook, execution_date=20220213T170402, start_date=20220213T170406, end_date=20220213T170406
[2022-02-13 17:04:07,039] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-13 17:04:07,105] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
